data/human/jobs.txt

```diff
@@ -3257,6 +3260,8 @@ mission "Bounty Hunting (Solo Marauder II)"
 			"combat rating" > 200
 			has "ships: Light Warship"
 			has "ships: Medium Warship"
+			has "ships: Light Carrier"
+			has "ships: Heavy Carrier"
```

Comment:
Heavy carriers are similar in size to heavy warships, so probably shouldn't be able to take on missions that heavy warships are blocked from.